### DISCUSSION TITLE
Favicon

### DISCUSSION THREAD

**Discussion by danicaso-vizzuality** [Vizzuality/mangrove-atlas#310] [Ideas]
Initially, we had a turquoise circle as a minimal version of the logo. Right now we have a full logo, including the name of the platform. Might we use a simple logo without text?

I believe I can not attach images to these messages in github, let me know if I'm missing how to do it. Meanwhile, I'll share a dropbox link: https://www.dropbox.com/s/hq1v9q3obq9cxhv/favicon.png?dl=0

**Comment by lyubovpencheva** [comment]
This is an old one.
